pkg/messageformat/lex.go

```diff
@@ -51,9 +52,9 @@ func lex(input string) *lexer {
 type lexer struct {
 	input        string
 	token        Token
+	prevToken    Token
```

Comment:
Perhaps *Token would be a more efficient way to check the contents of prevToken.

Reply:
explain ?